app/presenters/breadcrumbs.rb

```diff
@@ -0,0 +1,18 @@
+module Breadcrumbs
+  def breadcrumbs
+    return [] unless parent
+
+    direct_parent = parent
+    ordered_parents = []
+
+    while direct_parent
+      ordered_parents.unshift(
+        title: direct_parent["title"],
+        url: direct_parent["base_path"],
+      )
+      direct_parent = direct_parent["parent"] && direct_parent["parent"].first
```

Comment:
Parent is a nested hash of parents, this method traverses it until it reaches an item that does not have a parent.